.github/workflows/reuse-run-e2e-tests.yml

```diff
@@ -172,14 +172,14 @@ jobs:
           docker ps -a >> docker_logs/docker_ps.txt || true
           docker logs coordinator --since 1h &>> docker_logs/coordinator.txt || true
           docker logs prover --since 1h &>> docker_logs/prover.txt || true
-          docker logs prover-v3 --since 1h &>> docker_logs/prover-v3.txt; || true
+          docker logs prover-v3 --since 1h &>> docker_logs/prover-v3.txt || true
           docker logs shomei --since 1h &>> docker_logs/shomei.txt || true
           docker logs zkbesu-shomei --since 1h &>> docker_logs/zkbesu-shomei.txt || true
           docker logs shomei-frontend --since 1h &>> docker_logs/shomei-frontend.txt || true
           docker logs postman --since 1h &>> docker_logs/postman.txt || true
           docker logs traces-node --since 1h &>> docker_logs/traces-node.txt || true || true
```

Comment:
Nickpick (not your changes): `true || true`

Reply:
Thanks!